demo/DemoApp.EFDataExample/RulesEngineContext.cs

```diff
@@ -24,15 +27,28 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 entity.Ignore(b => b.WorkflowsToInject);
             });
 
+            //modelBuilder.Entity<Rule>()
```

Comment:
Thanks for this PR!

Can we remove the commented code if it is not needed?

Reply:
Oops, done!